gpflow/config/__config__.py

```diff
@@ -162,7 +220,7 @@ def set_default_positive_minimum(value: float):
 
 
 def set_default_summary_fmt(value: str):
-    formats = tabulate.tabulate_formats + ['notebook', None]
+    formats = tabulate.tabulate_formats + ["notebook", None]
```

Comment:
Why do you change the type of string quotes? I've never spelled it out explicitly anywhere but I've always thought of `'` for "string-constants" and `"` for "message-strings" - i.e. if it's something that should be exactly this, single-quotes, and if it's something that's just an example or wouldn't change behaviour if changed, double-quotes...

Reply:
I don't change it, the black does - https://black.readthedocs.io/en/stable/the_black_code_style.html#strings. `black` became a golden standard in the python world. Generally, I prefer double quotes and I don't know the language (except python and SQL) that represents strings with single quotes. For me as C/C++ programmer, it is even more confusing, single quotes used for char type.

Reply:
> I've never spelled it out explicitly anywhere but I've always thought of ' for "string-constants"

Btw, what you are describing for single quotes is an atom type. Sadly, python doesn't have it and python can't check it at compile or run time.